README.md

```diff
@@ -59,39 +59,85 @@ module "s3_anti_virus" {
+| lambda\_package                    | The name of the lambda package. Used for a directory tree and zip file. | `string`       | `"anti-virus"`            |    no    |
+| lambda\_s3\_bucket                 | The name of the S3 bucket used to store the Lambda builds.              | `string`       | n/a                       |   yes    |
+| lambda\_version                    | The version the Lambda function to deploy.                              | `string`       | n/a                       |   yes    |
+| name\_scan                         | Name for resources associated with anti-virus scanning                  | `string`       | `"s3-anti-virus-scan"`    |    no    |
+| name\_update                       | Name for resources associated with anti-virus updating                  | `string`       | `"s3-anti-virus-updates"` |    no    |
+| tags                               | A map of tags to add to all resources.                                  | `map(string)`  | `{}`                      |    no    |
 
 ## Outputs
 
-No output.
+| Name                                      | Description                                                    |
+| ----------------------------------------- | -------------------------------------------------------------- |
+| scan\_aws\_cloudwatch\_log\_group\_arn    | ARN for the Anti-Virus Scanning Cloudwatch LogGroup.           |
+| scan\_aws\_cloudwatch\_log\_group\_name   | The Anti-Virus Scanning Cloudwatch LogGroup name.              |
+| scan\_lambda\_function\_arn               | ARN for the Anti-Virus Scanning lambda function.               |
+| scan\_lambda\_function\_iam\_role\_arn    | Name of the Anti-Virus Scanning lambda role.                   |
+| scan\_lambda\_function\_iam\_role\_name   | Name of the Anti-Virus Scanning lambda role.                   |
+| scan\_lambda\_function\_name              | The Anti-Virus Scanning lambda function name.                  |
+| scan\_lambda\_function\_version           | Current version of the Anti-Virus Scanning lambda function.    |
+| update\_aws\_cloudwatch\_log\_group\_arn  | ARN for the Anti-Virus Definitions Cloudwatch LogGroup.        |
+| update\_aws\_cloudwatch\_log\_group\_name | The Anti-Virus Definitions Cloudwatch LogGroup name.           |
+| update\_lambda\_function\_arn             | ARN for the Anti-Virus Definitions lambda function.            |
+| update\_lambda\_function\_iam\_role\_arn  | ARN of the Anti-Virus Definitions lambda role.                 |
+| update\_lambda\_function\_iam\_role\_name | Name of the Anti-Virus Definitions lambda role.                |
+| update\_lambda\_function\_name            | The Anti-Virus Definitions lambda function name.               |
+| update\_lambda\_function\_version         | Current version of the Anti-Virus Definitions lambda function. |
```

Comment:
These changes will fail our build tests. We use `pre-commit` and `terraform-docs` which will revert all this whitespace when run. If you can do `brew install pre-commit` or `pip install pre-commit` followed by `pre-commit run -a` it will fix up the README.md as expected.